pygfx/objects/_base.py

```diff
@@ -400,6 +403,26 @@ def look_at(self, target: Vector3):
             self._q.set_from_rotation_matrix(self._m)
             self.rotation.premultiply(self._q.inverse())
 
+    def get_world_position(self):
+        self.update_matrix_world(update_parents=True, update_children=False)
+        self._v.set_from_matrix_position(self._matrix_world)
+        return self._v.clone()
+
+    def get_world_bounding_box(self, include_children=True):
+        self.update_matrix_world(update_parents=True, update_children=False)
```

Comment:
This causes a lot of unnecessary back and forth, can probably make it more efficient by only updating world matrices once